context/template.sparql

```diff
@@ -48,6 +51,17 @@ template :axioms(?prefix) {
        vann:preferredNamespacePrefix ?prefix .
 }
 
+template :changelog(?prefix) {
+    format { "<dt>%s</dt>" ?versionInfo }
+    format { "<dd>%s</dd>" ?changes }
+} where {
+    ?o a owl:Ontology ;
+         owl:versionInfo ?versionInfo ;
+         owl:priorVersion ?priorVersion ;
+         vann:changes ?changes .
+    # TODO: Call template with priorVersion URL
```

Comment:
After trying out a number of different strategies, I haven't really found a working approach to recursively call this template while using `priorVersion` as the new reference point. Do you maybe have an idea, @mahdanoura, how this could be done and whether this is possible at all using Sparkle Templates?

Reply:
I think I've now got this to work :) This required some additional adjustments in the JS-based render script, but eventually I managed to render the changelog from `owl:priorVersion`, `owl:versionInfo`, and `vann:changes`.

However, as there is no version 1.0.0 published yet, it is currently not possible to refer to the previous version. Thus, we might want to publish the placeholder HTML first or make this version the actual version 1.0.0.

Reply:
> After already trying out a couple of different strategies in the meantime, I haven't really found a working approach to recursively call this template while using `priorVersion` as the new reference point. Do you maybe have an idea, @mahdanoura, how this could be done and whether this is possible at all using SPARQL Templates?

To the best of my knowledge, it is not possible to achieve such a recursive behaviour with SPARQL templates.